libc/test/src/math/smoke/TotalOrderMagTest.h

```diff
@@ -0,0 +1,143 @@
+//===-- Utility class to test flavors of totalordermag ----------*- C++ -*-===//
```

Comment:
I omitted the word "different" in "test [different] flavors", unlike usual, so that `-*- C++ -*-` can fit.